src/Packagist/WebBundle/Resources/config/services.yml

```diff
@@ -26,14 +26,12 @@ services:
 
     packagist.user_repository:
         class: Packagist\WebBundle\Entity\UserRepository
-        factory_service: doctrine
-        factory_method: getRepository
+        factory: [@doctrine, getRepository]
```

Comment:
@alcohol ``@`` is reserved in the YAML spec. This is not new. But the Symfony parser was accepting it, so not following the spec.

@Soullivaneuh the Yaml parser has no concept of services. The YamlFileLoader expects a string starting with ``@`` to reference a service